### ISSUE TITLE
xcode 15 build fails with sentry-react-native 5.11.1 (RN 0.72.3)

### ISSUE THREAD

**Issue by leurs247** [getsentry/sentry-react-native#3366]
**OS:**
- [ ]  Windows
- [ x]  MacOS
- [ ]  Linux

**Platform:**
- [ x]  iOS
- [ ]  Android

**SDK:**
- [ x]  `@sentry/react-native` (>= 1.0.0)
- [ ]  `react-native-sentry` (<= 0.43.2)

**SDK version:** 5.11.1

**`react-native` version:** 0.72.3

**Are you using Expo?**
- [ ]  Yes
- [ x]  No

**Are you using sentry.io or on-premise?**
- [ x]  sentry.io (SaaS)
- [ ]  on-premise

I have installed sentry using ``npx @sentry/wizard@latest -s -i reactNative``. It is all installed, in App.js is see sentry code.

When building with xcode 15, my build fails. I get the following error:
````
Command PhaseScriptExecution failed with a nonzero exit code
````

![CleanShot 2023-10-29 at 11 15 55@2x](https://github.com/getsentry/sentry-react-native/assets/58610521/0d6eb2a4-720b-4853-b2f4-725b906ba29f)

**Bundle react native code and images:**
```
export SENTRY_PROPERTIES=sentry.properties
export EXTRA_PACKAGER_ARGS="--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map"
export EXTRA_PACKAGER_ARGS="--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map"
set -e

WITH_ENVIRONMENT="../node_modules/react-native/scripts/xcode/with-environment.sh"
REACT_NATIVE_XCODE="../node_modules/react-native/scripts/react-native-xcode.sh --force-foreground"

/bin/sh -c "$WITH_ENVIRONMENT \"\"../node_modules/@sentry/cli/bin/sentry-cli react-native xcode $REACT_NATIVE_XCODE\"\""


/bin/sh -c "$WITH_ENVIRONMENT ../node_modules/@sentry/react-native/scripts/collect-modules.sh"
```

As you can see from the screenshot, the sentry-cli code is executed but it just gives the standard output. And after this output I get the error.

What could possibly be wrong? I don't get an error like file not found. If I change the path in my build scripts, it fails and I get the error "not found" so at least the cli is found.

**Comment by krystofwoldrich** [comment]
Hi,
thank you for the message.

It looks like there are double quotes in your script, which might cause the issue.

Can you try this:
```
export SENTRY_PROPERTIES=sentry.properties
export EXTRA_PACKAGER_ARGS="--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map"
set -e

WITH_ENVIRONMENT="../node_modules/react-native/scripts/xcode/with-environment.sh"
REACT_NATIVE_XCODE="../node_modules/react-native/scripts/react-native-xcode.sh --force-foreground"

/bin/sh -c "$WITH_ENVIRONMENT \"../node_modules/@sentry/cli/bin/sentry-cli react-native xcode $REACT_NATIVE_XCODE\""

/bin/sh -c "$WITH_ENVIRONMENT ../node_modules/@sentry/react-native/scripts/collect-modules.sh"
```

**Comment by Nehal-Zeller** [comment]
@krystofwoldrich  I'm also facing the same issue, tried your script but that didn't help.

Xcode: 15.0.1
"react": "18.2.0",
"react-native": "0.71.14",

script which was working earlier before the Xcode 15 update
`set -e
export SENTRY_PROPERTIES=sentry.properties
export EXTRA_PACKAGER_ARGS="--sourcemap-output $DERIVED_FILE_DIR/main.jsbundle.map"

WITH_ENVIRONMENT="../node_modules/react-native/scripts/xcode/with-environment.sh"
REACT_NATIVE_XCODE="../node_modules/react-native/scripts/react-native-xcode.sh"
SENTRY_CLI_PATH="../node_modules/@sentry/cli/bin/sentry-cli"

/bin/sh -c "$WITH_ENVIRONMENT \"$SENTRY_CLI_PATH react-native xcode $REACT_NATIVE_XCODE\""
/bin/sh ../node_modules/@sentry/react-native/scripts/collect-modules.sh`

Error:
<img width="729" alt="Screenshot 2023-11-02 at 8 44 24 am" src="https://github.com/getsentry/sentry-react-native/assets/134464804/65485cce-5b36-47bb-8d2d-f4099fc04a90">

**Comment by Nehal-Zeller** [comment]
Issue is resolved on updating sentry to 5.12.0, thanks!!

**Comment by krystofwoldrich** [comment]
@Nehal-Zeller Thanks for the message, closing the issue.
